### DISCUSSION TITLE
New Logo Proposal/Suggestion

### DISCUSSION THREAD

**Discussion by SecularSteve** [snuffyDev/Beatbump#193] [Ideas]
Long story short, the old logo seems a bit dated these days.
Considering that Apple's planning on scrapping and archiving their iconic bud design, it's inevitable.

Thus I propose a new design. Made the varieties myself in Inkscape:

![BeatBump New](https://github.com/snuffyDev/Beatbump/assets/33793273/357abcc3-0f61-45c9-837f-9734c5ebeec6)
![BeatBump New](https://github.com/snuffyDev/Beatbump/assets/33793273/a469ba6c-ea7a-41f7-97ca-538775e46f5b)

Reelased under the CC-BY SA 4.0
ZIP contains SVG and PNG.
[BeatBump New.zip](https://github.com/snuffyDev/Beatbump/files/12407784/BeatBump.New.zip)

**Comment by SecularSteve** [comment]
P.S. You may make your own suggestions if you want to as well.
Personally I'd be interested in what all of you may come up with.

**Comment by snuffyDev** [comment]
My apologies for not replying! A logo upgrade would be amazing how can I contact you for further changes (discord?)

**Reply by SecularSteve** [reply]
Discord is no issue.
Or simply mail.
